Ex 2/main3.js

```diff
@@ -0,0 +1,7 @@
+function c() {
+  var sum = 0;
+  let x = parseInt(document.getElementById("a").value);
+  let y = parseInt(document.getElementById("b").value);
+  sum = x+y;
```

Comment:
Ko khai báo biến, 5k
```suggestion
  const sum = x + y;
```
Và dấu cách đâu